### ISSUE TITLE
What do the features mean in Tox21?

### ISSUE THREAD

**Issue by rmurphy2718** [deepchem/deepchem#2420]
## ❓ Questions & Help

When we create a data loader,
```
tasks, datasets, transformers = dc.molnet.load_tox21(featurizer='GraphConv')
```
there are several atom and pairwise features.  I cannot find it clearly in the docs what these features mean/represent.  Where is it listed in the docs?

Thank you

**Comment by rbharath** [comment]
Copying over the answer from gitter for record keeping:

Check out https://deepchem.readthedocs.io/en/latest/api_reference/data.html?highlight=data%20classes#graph-data
These are ConvMol objects which are produced by ConvMolFeaturizer. For more details about the atom features check out https://deepchem.readthedocs.io/en/latest/api_reference/featurizers.html#convmolfeaturizer

**Comment by rmurphy2718** [comment]
Sorry for the duplicate, I later decided this may be of general interest to future users.

Can I clarify that the "Featurizer" is adding feature vectors to the vertices/atoms?  I became confused because the [MoleculeNet paper ](https://arxiv.org/abs/1703.00564) refers to _featurization_ as the process of converting a molecule to a vector, e.g. with ECFP.  

But here, am I correct that `ConvMolFeaturizer` adds attributes to vertices _before_ the Duvenaud convolutions?
```
def _featurize(self, mol):
    """Encodes mol as a ConvMol object."""
    # Get the node features
    idx_nodes = [(a.GetIdx(),
                  np.concatenate((atom_features(
                      a, use_chirality=self.use_chirality),
                                  self._get_atom_properties(a))))
                 for a in mol.GetAtoms()]
``` 

So, to sum up, what MoleculeNet calls featurization is what Deep Learning folks call "learning a representation" (or embedding), but in DeepChem features really do refer to the attributes of atoms passed to the model?

Thank you!

**Comment by rbharath** [comment]
Perhaps a better definition would be that featurization is the process of converting a molecule to a representation suitable for learning. `ConvMolFeaturizer` converts molecules into molecules graphs where each atom is annotated with a local descriptor vector based on simple chemical attributes. The `GraphConvModel` then creates an embedding (the learned representation) that converts this input graph into a vector based on provided training data. I hope that clarifies!
